main.go

```diff
@@ -92,6 +92,9 @@ func main() {
 		esClusterInfoInterval = kingpin.Flag("es.clusterinfo.interval",
 			"Cluster info update interval for the cluster label").
 			Default("5m").Duration()
+		esExportLicense = kingpin.Flag("es.license",
```

Comment:
Using the collector interface (collector/collector.go), this isn't necessary. A new flag will automatically be created (`--[no]collector.cluster_license`)